src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {
+    public Figure getRandomFigure() {
+        Figure temp = null;

```

Comment:
```suggestion
        Figure figure = null;

```

not informative variable names